internal/db/postgres/context/pg_catalog.go

```diff
@@ -351,6 +361,7 @@ func BuildTableSearchQuery(
 		   n.nspname                              as "Schema",
 		   c.relname                              as "Name",
 		   pg_catalog.pg_get_userbyid(c.relowner) as "Owner",
+		   pg_catalog.pg_relation_size(c.oid)     as "Size",
```

Comment:
@wwoytenko did you consider using `pg_table_size` here, since that includes TOAST?

This PR definitely improves the situation, and most of our big tables are processes first, but the biggest of them  all still ends up being the bottleneck because most of the data is in TOAST

Reply:
Yep, important remark. I will renew the size calculation in the new PR

Reply:
@janmeier should be fixed in #71